deploy/olm-catalog/bundle.Dockerfile.ci-index-image-upgrade

```diff
@@ -0,0 +1,28 @@
+FROM quay.io/fedora/fedora:33-x86_64 AS builder
```

Comment:
Is this a copy of `deploy/index-image/bundle.Dockerfile.ci-index-image-upgrade`? why do we need both?

Reply:
Because all the contents under `deploy/index-image` are being deleted and recreated in `build-manifests.sh`